tests/integration/features.go

```diff
@@ -824,3 +824,37 @@ func CheckFluentdEventsInstall(builder *features.FeatureBuilder) *features.Featu
 				return ctx
 			})
 }
+
+func CheckOtellogsMetricsDisabled(builder *features.FeatureBuilder) *features.FeatureBuilder {
+	return builder.
+		Assess("otelllogs metrics are disabled",
+			func(ctx context.Context, t *testing.T, envConf *envconf.Config) context.Context {
+				releaseName := ctxopts.HelmRelease(ctx)
+				var services []corev1.Service
+				require.Eventually(t, func() bool {
+					services = terrak8s.ListServices(t, ctxopts.KubectlOptions(ctx), v1.ListOptions{
+						LabelSelector: fmt.Sprintf("app=%s-sumologic-otelcol-logs-collector", releaseName),
+					})
+
+					return len(services) == 0
+				}, waitDuration, tickDuration)
+				return ctx
+			})
+}
+
+func CheckOtellogsMetricsEnabled(builder *features.FeatureBuilder) *features.FeatureBuilder {
+	return builder.
+		Assess("otelllogs metrics are enabled",
+			func(ctx context.Context, t *testing.T, envConf *envconf.Config) context.Context {
+				releaseName := ctxopts.HelmRelease(ctx)
+				var services []corev1.Service
+				require.Eventually(t, func() bool {
+					services = terrak8s.ListServices(t, ctxopts.KubectlOptions(ctx), v1.ListOptions{
+						LabelSelector: fmt.Sprintf("app=%s-sumologic-otelcol-logs-collector", releaseName),
+					})
+
+					return len(services) == 1
+				}, waitDuration, tickDuration)
+				return ctx
+			})
+}
```

Comment:
Please make it gennric

```
func checkOtellogsMetricsServiceCount(ctx context.Context, t *testing.T, envConf *envconf.Config, count int) context.Context {
			return func(ctx context.Context, t *testing.T, envConf *envconf.Config) context.Context {
				releaseName := ctxopts.HelmRelease(ctx)
				var services []corev1.Service
				require.Eventually(t, func() bool {
					services = terrak8s.ListServices(t, ctxopts.KubectlOptions(ctx), v1.ListOptions{
						LabelSelector: fmt.Sprintf("app=%s-sumologic-otelcol-logs-collector", releaseName),
					})

					return len(services) == count
				}, waitDuration, tickDuration)
				return ctx
			}
}
```

```suggestion
			checkOtellogsMetricsServiceCount(ctx, t, envConf, 1)
}
```

Reply:
I changed tests to template tests

Reply:
Now there is no changes for features.go :fearful:

Reply:
To test if service is available or not and if it has proper form we can use template tests instead of integration tests.
Do you think that integration tests are needed also?

Reply:
Ou, I misunderstood the context of `template`

I think we can use `template` tests, but in current tests form empty files don't test anything

Could you try if the following snippets are correct?
```
---
# Source: sumologic/templates/logs/collector/otelcol/service.yaml
```